gui/wxpython/wxgui.py

```diff
@@ -137,16 +182,23 @@ def main(argv=None):
         argv = sys.argv
     try:
         try:
-            opts, args = getopt.getopt(argv[1:], "hw:", ["help", "workspace"])
+            opts, args = getopt.getopt(
+                argv[1:],
+                "hwr:",
+                ["help", "workspace", "recent_files"],
+            )
         except getopt.error as msg:
             raise Usage(msg)
     except Usage as err:
         print(err.msg, file=sys.stderr)
         print(sys.stderr, "for help use --help", file=sys.stderr)
         printHelp()
 
-    workspaceFile = process_opt(opts, args)
-    app = GMApp(workspaceFile)
+    parsed_args = process_opt(opts, args)
+
+    print_wxgui_component_recent_files(args=parsed_args)
```

Comment:
This is a bit unexpected, the function should be called only if the option is active.